www-client/weboob/metadata.xml

```diff
@@ -5,9 +5,22 @@
 		<email>patrick@gentoo.org</email>
 		<name>Patrick Lauer</name>
 	</maintainer>
+	<maintainer type="person">
+		<email>laurent@bachelier.name</email>
+		<name>Laurent Bachelier</name>
+	</maintainer>
```

Comment:
That is not going to work. Either you maintain it, or Laurent is filing this PR.